pkg/cmd/server/origin/controller.go

```diff
@@ -69,21 +69,23 @@ func (c *MasterConfig) NewOpenShiftControllerPreStartInitializers() (map[string]
 		}
 		saTokens.ServiceServingCA = append(saTokens.ServiceServingCA, serviceServingCA...)
 	}
-	ret["serviceaccount-tokens"] = saTokens.RunController
+	// this matches the upstream name
+	ret["serviceaccount-token"] = saTokens.RunController
 
 	return ret, nil
 }
 
 func (c *MasterConfig) NewOpenshiftControllerInitializers() (map[string]controller.InitFunc, error) {
 	ret := map[string]controller.InitFunc{}
 
+	// TODO this overrides an upstream controller, so move this to where we initialize upstream controllers
 	serviceAccount := controller.ServiceAccountControllerOptions{
 		ManagedNames: c.Options.ServiceAccountConfig.ManagedNames,
 	}
 	ret["serviceaccount"] = serviceAccount.RunController
 
-	ret["serviceaccount-pull-secrets"] = controller.RunServiceAccountPullSecretsController
-	ret["origin-namespace"] = controller.RunOriginNamespaceController
+	ret["openshift.io/serviceaccount-pull-secrets"] = controller.RunServiceAccountPullSecretsController
```

Comment:
> openshift.io/controllers/ ?
> 
> how about some generator with:
> 
> ret.AddOpenShiftController(controller.RunServiceAccount...)
> ret.AddKubeController(...)
> 
> return ret.InitializersMap()
> too much?

I think we actually end up creating a method to create `NewKubeControllerInitializers` and then one method that does `NewAllControllerInitializers` or something.  That will help us prep for splitting apart.